test/component_test.go

```
@@ -0,0 +1,126 @@
+	assert.Equal(s.T(), "us-east-2", cloudtrailHomeRegion)
+	assert.Equal(s.T(), *trail.HomeRegion, cloudtrailHomeRegion)
+
+	assert.True(s.T(), *trail.IsOrganizationTrail)
+
+	s.DriftTest(component, stack, nil)
+}
+
+func (s *ComponentSuite) TestEnabledFlag() {
+	const component = "cloudtrail/disabled"
+	const stack = "default-test"
+	const awsRegion = "us-east-2"
```

golangci-lint (1.62.2)

const `awsRegion` is unused

(unused)